husarion_ugv_hardware_interfaces/src/robot_system/robot_driver/roboteq_data_converters.cpp

```diff
@@ -102,7 +102,7 @@ FaultFlag::FaultFlag()
     "overvoltage",
     "undervoltage",
     "short_circuit",
-    "emergency_stop",
+    "emergency_stop (torque disabled)",  // Roboteq E-stop differs from ROS E-stop
```

Comment:
```suggestion
    "emergency_stop",
```
Not sure if that information is needed. This field is under `robot_driver -> fault_flags -> emergency_stop` should be obvious it is different than system e-sotp

Reply:
As I wrote above, it would not require the user to have knowledge of roboteq and I would avoid name collisions. Especially since e_stop means something different for the roboteq driver than for the ros driver. Currently, the warning only occurs during e_stop and there is clear information that it is caused by turning off torque.